controllers/actions_controller.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Actions Controller.
+ *
+ * Manages the top level actions queue for the gateway (i.e. not actions on
+ * Things but on the gateway itself.)
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+var express = require('express');
+var Things = require('../models/things');
+var AdapterManager = require('../adapter-manager');
+
+var ActionsController = express.Router();
+
+ActionsController.post('/', function (request, response) {
+  if (!request.body.name) {
+    response.status(400).send();
+    return;
+  }
+  switch(request.body.name) {
+    case 'pair':
+      console.log('Actions Controller requested addNewDevice');
+      AdapterManager.addNewDevice();
+      //TODO: Create an action resource in the database and return its ID
```

Comment:
I'm not sure that these should be persisted in the database.

If we need to persist something then we probably want persistable actions versus non-persistable actions. Pairing for both ZigBee and ZWave would be in the "non-persistable" category, and would just need an array/dict in the server app.